File: dvid_to_neuprint/remove_0_label_syns.pl
#!/usr/bin/perl

use strict;

my $file = $ARGV[0];

print "synId,x,y,z,type,confidence,toplevel_roi,sub1_roi,sub2_roi,sub3_roi,body\n";
open(IN,"$file");
while (my $line = <IN>) {
    if ($line =~ /^\d+/) {
	chomp($line);	
	my @data = split(/\,/,$line);
    
	if ($data[10] != 0) {
	    print "$line\n";
	}
    }
}
close(IN);

File: dvid_to_neuprint/remove_0_label_syns.py
#!/bin/env
# ------------------------- imports -------------------------
import sys
import os

if __name__ == '__main__':
    synapse_body_map_file = sys.argv[1]

    synCount = 0
    print("synId,x,y,z,type,confidence,toplevel_roi,sub1_roi,sub2_roi,sub3_roi,body")
    synapseBodyMapList = open(synapse_body_map_file,'r')
    for line in synapseBodyMapList:
        if line[0].isdigit():
            data_str = line.rstrip('\n')
            synapseBodyMapData = data_str.split(',')
            if int(synapseBodyMapData[10]) != 0:
                print(data_str)
